db/seeds.rb

```diff
@@ -7,8 +7,35 @@
 #   Mayor.create(name: 'Emanuel', city: cities.first
 
 user = User.create(
-  username: username
+  username: "username",
   email: "user@email.com",
   password: "password",
   password_confirmation: "password"
 )
+
+bucket_list = BucketList.create(
+  title: 'Bucket List 1',
+  is_public: true
+)
+
+adventure_1 = Adventure.create(
+  name: 'do things'
+)
+
+adventure_2 = Adventure.create(
+  name: 'do other things'
+)
+
+adventure_bucket_join_1 = BucketListAdventure.create(
+  adventure_id: 1,
+  bucket_list_id: 1,
+  is_achieved: false,
+  notes: "blah blah blah"
+)
+
+adventure_bucket_join_2 = BucketListAdventure.create(
```

Comment:
Useless assignment to variable - `adventure_bucket_join_2`. Did you mean `adventure_bucket_join_1`?